src/draw/lv_draw.h

```diff
@@ -119,8 +119,10 @@ typedef struct _lv_draw_unit_t {
      * @return              >=0:    The number of taken draw task:
      *                                  0 means the task has not yet been completed.
```

Comment:
In the SW dispatcher it it not the case as it supports all the tasks.

But in the GPU dispatcher you will have something like this:
```
    /* Try to get an ready to draw. */
    lv_draw_task_t * t = lv_draw_get_next_available_task(layer, NULL, DRAW_UNIT_ID_GPU);

    /* Return 0 is no selection, some tasks can be supported by SW unit only. */
    if(t == NULL)
        return 0;
```

Reply:
> In the SW dispatcher it it not the case as it supports all the tasks.
> 
> But in the GPU dispatcher you will have something like this:
> 
> ```
>     /* Try to get an ready to draw. */
>     lv_draw_task_t * t = lv_draw_get_next_available_task(layer, NULL, DRAW_UNIT_ID_GPU);
> 
>     /* Return 0 is no selection, some tasks can be supported by SW unit only. */
>     if(t == NULL)
>         return 0;
> ```
> 
> Returning 0 will allow the SW dispatcher to run, and:
> 
> * if task taken it will return 1
> * if no more tasks available, it will return -1 and will break the while loop.

I have deleted error `break` sentence. Please just return -1 when there are no tasks to be processed. You can check it again.

Reply:
Oh, my bad about the -1. Sorry.

But still, what shall the GPU dispatcher return in the above code?

Reply:
return -1